honeybee_radiance/properties/dynamic.py

```diff
@@ -0,0 +1,412 @@
+# coding=utf-8
+"""Object representing a group of sub-faces or shades that change states in unison."""
+from __future__ import division
+
+from ..state import RadianceShadeState, RadianceSubFaceState
+from ..geometry import Polygon
+from ..modifier.material import BSDF
+from ..lib.modifiers import white_glow
+
+from honeybee.typing import valid_rad_string
+from honeybee.shade import Shade
+from honeybee.aperture import Aperture
+from honeybee.door import Door
+from honeybee.boundarycondition import Surface
+
+import os
+
+
+class DynamicShadeGroup(object):
+    """Object representing a group of dynamic shades that change states in unison.
+
+    Note that this object is not intended to set any Model properties. It is only
+    used by the Model to collect objects with the same dynamic_group_identifier
+    and export a common set of radiance files from them.
+
+    Args:
+        identifier: Text string for a unique Group ID. Must not contain spaces
+            or special characters. This will be used to identify the object across
+            a model and in the exported Radiance files.
+        dynamic_objects: An array of Shades that are contained within the group.
+
+    Properties:
+        * identifier
+        * dynamic_objects
+        * state_count
+        * is_opaque
+        * is_indoor
+        * states_json_list
+    """
+
+    __slots__ = ('_identifier', '_dynamic_objects', '_state_count')
+
+    def __init__(self, identifier, dynamic_objects):
+        """Initialize DynamicShadeGroup."""
+        # set identifier
+        self._identifier = valid_rad_string(identifier)
```

Comment:
Shouldn't we get the identifier from dynamic_objects instead of setting it up here?

Reply:
We do. See the top docstring of this class:
```
Note that this object is not intended to set any Model properties. It is only
used by the Model to collect objects with the same dynamic_group_identifier
and export a common set of radiance files from them.
```
If it makes it clearer that the user isn't supposed to be initializing these classes themselves (without using the Model to do so), I can make this module hidden. I just made it public since it would be nice to have the sphinx docs generated to show the user what they can do with these objects when they get them from a Model.

Reply:
I agree that having it as part of the documentation is good. Let's see if it will be an issue.